NovaPushNotificationServiceExtension/Handlers/PushNotificationHandlersFactory.swift

```diff
@@ -1,10 +1,12 @@
 import Foundation
 import SoraFoundation
 
+typealias PushNotificationHandleResult = Result<NotificationContentResult, PushNotificationsHandlerErrors>
```

Comment:
I would make semantic more concrete then. Instead of result we could have enum: modified(NotificationContent) (the original content is changed), filteredOut (no need to display the notification), original(Error?) (display original content and log the error)